src/runner.js

```diff
@@ -132,11 +132,14 @@ exports.run = function(config) {
     // establish ngrok tunnel and resolve ngrok url
```

Comment:
Change comment to `establish tunnel (ngrok or sauce connect)`

Reply:
changed